filebeat/crawler/prospector.go

```diff
@@ -98,18 +111,43 @@ func (p *Prospector) Run(wg *sync.WaitGroup) {
 }
 
 func (p *Prospector) Stop() {
+	logp.Debug("prospector", "Stopping %v harvesters.", len(p.harvesters))
+	for _, h := range p.harvesters {
+		go h.Stop()
+	}
+	logp.Debug("prospector", "stopp called on all harvesters")
```

Comment:
`s/stopp/stop/`, but I would prefer to see something like `"Waiting for %d harvesters to stop", len(p.harvesters)`.